sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/data/HistogramPointDataValidations.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.data;
+
+import io.opentelemetry.sdk.metrics.data.HistogramPointData;
+import java.util.List;
+
+/**
+ * Validations for {@link HistogramPointData}.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+final class HistogramPointDataValidations {
+
+  private HistogramPointDataValidations() {}
+
+  static boolean isStrictlyIncreasing(List<Double> xs) {
```

Comment:
I noticed that this message returns a boolean while `validateBoundaryEdgesAreNotInfinite` throws. Can we make them consistent and throw in both places? It appears that both usages of `isStrictlyIncreasing` throw the same exception and error message if this returns false.

Reply:
Good point. I made them the same now.